pkgs/applications/networking/testssl/default.nix

```diff
@@ -12,21 +10,21 @@ in stdenv.mkDerivation rec {
     owner = "drwetter";
     repo = "testssl.sh";
     rev = "v${version}";
-    sha256 = "0pfp7r4jhvkh06vawqlvq7vp4imwp6dpq6jx8m0k3j85ywwp45pd";
+    sha256 = "0zgj9vhd8fv3a1cn8dxqmjd8qmgryc867gq7zbvbr41lkqc06a1r";
   };
 
   patches = [ ./testssl.patch ];
 
   postPatch = ''
     substituteInPlace testssl.sh                                               \
-      --replace /bin/pwd                    ${pwdBinPath}                      \
+      --replace /bin/pwd                    ${coreutils}/bin/pwd               \
```

Comment:
I can't find `/bin/pwd` in testssl.sh

Reply:
It's in there, when I do `grep 'bin/pwd' result/bin/testssl.sh` I get the following output:
```
          cwd="$(/nix/store/jy9knxp7nmw80jkf932axrs1b4p9k4hi-coreutils-8.29/bin/pwd)" || \
     if [[ "$openssl_location" =~ $(/nix/store/jy9knxp7nmw80jkf932axrs1b4p9k4hi-coreutils-8.29/bin/pwd)/bin ]]; then
```

On the patched one.

Reply:
Line 11404 and 11406 it seems to be :p

Reply:
Ah, sorry. I was looking at the development version.

Reply:
However, I think that's a bit inconsistent: There are other places where `$(pwd)` or `$(printf)` is used. Maybe we should use `makeWrapper` to add `coreutils` to the path and replace `/bin/pwd` by `pwd`?

Reply:
I was told before that people rather patch bash scripts than to have wrappers for them, since it's easy to patch. But I've added a `--replace` for `$(pwd)` as well

Reply:
And what about `printf`, `basename`, `kill`, ...? I think in this case a wrapper is definitely the simplest solution.

Reply:
I also see `hostname` from `nettools` used